apps/coscrad-frontend/src/components/header/header.tsx

```diff
@@ -33,15 +34,29 @@ export const Header = (): JSX.Element => {
      * TODO: Move data (ConfigurableContentContext) out of presenter
      */
     const { siteTitle } = useContext(ConfigurableContentContext);
+    const { organizationLogoUrl } = useContext(ConfigurableContentContext);
 
     return (
         <StyledAppBar data-testid={'header'}>
             <StyledToolbar>
-                <StyledSiteTitle>
-                    <Link to="/">
-                        <Typography variant="h1">{siteTitle}</Typography>
-                    </Link>
-                </StyledSiteTitle>
+                <Link to="/">
+                    <Box style={{ display: 'flex', alignItems: 'center' }}>
+                        <StyledSiteTitle
+                            sx={{ marginRight: '20px', display: 'flex', alignItems: 'center' }}
+                        >
+                            <StyledImage
+                                sx={{ width: '30px' }}
+                                src={organizationLogoUrl}
+                                alt={siteTitle}
+                            />
+                        </StyledSiteTitle>
+                        <StyledSiteTitle>
+                            <Typography fontSize={'calc(18px + 0.5vw) !important'} variant="h1">
```

Comment:
Very slick!!  I checked it out on iPhone 5, so the smallest phone I think we'd consider (or not?) and there is a bit of overlap (see pasted screenshot), so I tried a numbers tweak that seemed to work: `calc(16px + 0.7vw)`

<img width="340" alt="image" src="https://github.com/COSCRAD/coscrad/assets/63014783/682a31e6-3fc7-4c16-8669-02b881a9b5f1">